datasets.toml

```diff
@@ -27,6 +27,18 @@ version=1.0
   url="https://osf.io/gnyt4/"
 
 
+[[datasets]]
+
+  key="Bakker-Jacobs_2021"
```

Comment:
OpenAlex reports 2022 as the publication year. There has been a lot of debate in the OpenAlex community on the "publication year". I think we used the OpenAlex reported year in our other datasets.